### PR TITLE
Add log softmax to model_logits

### PR DESCRIPTION
**Pull request by norabelrose**
Ensures that the language model probabilities are normalized across pseudo-labels

### CODE REVIEW

**Review by AlexTMallen — APPROVED** [review]
tested and lgtm
